src/style.css

```diff
@@ -1 +1,77 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
```

Comment:
instead @import for fonts use @font-face

Reply:
i can't connect using @font-face because google fonts doesn't support the format for <font-format>= [<string> | collection | embedded-opentype | opentype
  | svg | truetype | woff | woff2 ]